main.go

```diff
@@ -61,12 +69,104 @@ func main() {
 		}
 	}
 
+	if autoSetup && runtime.GOOS == "linux" {
```

Comment:
We can use build tags here instead of checking it at the runtime.

Reply:
How do I do this?

I understand you can do 
```go
// +build <GOOS>
```
But that seems to cover the entire file